### PR TITLE
docker-machine-parallels 1.1.1

### PR DESCRIPTION
**Pull request by kad**
- Compatibility with Docker Machine v0.5.5+
- Add arguments --version and --help for plugin binary

### CODE REVIEW

File: Library/Formula/docker-machine-parallels.rb [hunk 1]
```diff
@@ -24,12 +24,17 @@ class DockerMachineParallels < Formula
   end
 
   go_resource "github.com/docker/machine" do
-    # Docker Machine v0.5.1 release
-    url "https://github.com/docker/machine.git", :revision => "7e8e38e1485187c0064e054029bb1cc68c87d39a"
+    # Docker Machine v0.5.5 release
+    url "https://github.com/docker/machine.git", :revision => "02c4254cb4c93a4bbb5dc4ca0467abeb12d72546"
```
Comment (mikemcquaid) [inline Library/Formula/docker-machine-parallels.rb]: Maybe use a `:tag` argument so you don't need the comment?

File: Library/Formula/docker-machine-parallels.rb [hunk 2]
```diff
@@ -24,12 +24,17 @@ class DockerMachineParallels < Formula
...
   end
 
   go_resource "golang.org/x/crypto" do
-    url "https://go.googlesource.com/crypto.git", :revision => "beef0f4390813b96e8e68fd78570396d0f4751fc"
+    url "https://go.googlesource.com/crypto.git", :revision => "f18420efc3b4f8e9f3d51f6bd2476e92c46260e9"
+  end
+
+  go_resource "github.com/codegangsta/cli" do
+    # v1.2.0 release
+    url "https://github.com/codegangsta/cli.git", :revision => "565493f259bf868adb54d45d5f4c68d405117adf"
```
Comment (mikemcquaid) [inline Library/Formula/docker-machine-parallels.rb]: Maybe use a `:tag` argument so you don't need the comment?

**Comment by kad** [comment]
:revision in theory more explicit, as tags is something that can be re-created/moved to point to another sha1. But if you'd prefer tags, I can change, no problem with that.

**Comment by DomT4** [comment]
> :revision in theory more explicit, as tags is something that can be re-created/moved to point to another sha1.

We tend to use both when possible to ensure the tag doesn't move around, and to make it easier for people to know what the `revision` is pointing at.

**Comment by mhenrixon** [comment]
:+1:

**Comment by kad** [comment]
ok. having both :tag and :revision make sense. will update PR later tonight.

**Comment by kad** [comment]
@michaelsanford @DomT4 updated according to feedback. ( included :tag in addition to :revision )

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**